app/backend/src/User/schemas/User.ts

```diff
@@ -0,0 +1,34 @@
+import { z } from 'zod';
+
+const userZodSchema = z.object({
+  id: z.any().optional(),
```

Comment:
what is the purpose of this field?

Reply:
in mongodb i created a schema in zod that also infer as a type to create a database model. So, to create an object type User we dont need the id, right? But, as the PUT request needs some identifier and theyre sharing the same schema/type, i created this optional field that it scales well for futures finds